app/build.gradle

```diff
@@ -21,6 +21,8 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'com.android.support:design:27.1.1'
```

Comment:
This lines of code are the same in 2 different PRs?     mmm.... can we have a better solution? maybe erasing if they are not really necessary or making this PR dependable from another branch instead of Development....  by the way!!! Please make this PR to be raised against Development instead of master